Dockerfile

```diff
@@ -148,7 +148,8 @@ RUN apt-get update
 
 # Link FreeRDP plugins into proper path
 RUN ${PREFIX_DIR}/bin/link-freerdp-plugins.sh \
-        ${PREFIX_DIR}/lib/freerdp2/libguac*.so
+        ${PREFIX_DIR}                         \
+        /lib/freerdp2/libguac*.so
```

Comment:
The guacamole-server build automatically locates the proper directory for FreeRDP plugins and installs its plugins there. Perhaps it would be better to do away with `link-freerdp-plugins.sh` entirely and avoid this problem going forward?

Reply:
My initial understanding was that setting `LD_LIBRARY_PATH` to `/usr/local/guacamole/lib` would be sufficient as everything was available under that path, including the FreeRDP plugins, but if we skip the `link-freerdp-plugins.sh`, `guacd` complains that it can't load `"guac-common-svc" from /usr/lib/x86_64-linux-gnu/freerdp2/` so that's why I tried to adapt the `link-freerdp-plugins.sh`, but I'm happy to help if we can get rid of it and simplify this process.

> The guacamole-server build automatically locates the proper directory for FreeRDP plugins and installs its plugins there

I'm not sure to understand if you mean that it must already work as-is without `link-freerdp-plugins.sh` (which doesn't seems to be the case), or if it means that we need to adapt something based on what has been done during the build process (if that's the case could you give me pointers?).